packages/transaction-manager/test/unit/utils/test-data.ts

```diff
@@ -79,6 +105,43 @@ export const fakeDecryptionProvider: DecryptionProviderTypes.IDecryptionProvider
   supportedMethods: [EncryptionTypes.METHOD.ECIES],
 };
 
+export const fakeEpkCipherProvider: CipherProviderTypes.ICipherProvider & {
```

Comment:
If you make this an IEpkCipherProvider type, you wouldn't have to re-define so many of the methods.

Reply:
Actually, you can just make a local class that implements ICipherProvider, like you did for the `FakeLitProtocolProvider` below